documentation/modules/exploit/linux/local/exim4_deliver_message_priv_esc.md

```diff
@@ -0,0 +1,85 @@
+# Vulnerable Application
+
+Exim 4.87 - 4.91 Local Privilege Escalation
+
+This module implements a flaw found in Exim versions 4.87 to 4.91 (inclusive). Improper validation of recipient address in deliver_message() function in /src/deliver.c may lead to remote command execution (CVE-2019-10149).
```

Comment:
`exploits` would likely make more sense than `implements`. It is also tradition.

```
# grep -rn "This module exploits" modules/ | wc -l
1601
# grep -rn "This module implements" modules/ | wc -l
6
# grep -rn "This module implements" modules/exploits/ | wc -l
2
```